var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -25,26 +25,27 @@ class Neovim(CMakePackage):
     version('0.2.0', sha256='72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5')
 
     depends_on('cmake@3.0:', type='build')
+    depends_on('git@1.8.4:', type='build', when='@stable,master')
```

Comment:
Is this only needed to download the package? In that case I wouldn't bother making this an explicit dependency